src/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum AssetHash {
```

Comment:
I would rename it `Market` to be more explicit in what it is. 

We will fetch it from API using the `Markets()` endpoint

Reply:
Fixed this.